README.md

```diff
@@ -14,12 +13,13 @@ If you want to set up a Virtual Schema for a different database system, please h
 ### Information for Users
 
 * [PostgreSQL dialect](doc/dialects/postgresql.md)
-* [User Guide][user-guide]
+* [Generic Virtual Schema User Guide][user-guide]
+* [List of supported capabilities](doc/generated/capabilities.md)
```

Comment:
I would move it to information for developers

Reply:
I think it's important for uses. It's important information for understanding query performance